tests/syntax/test_modular.py

```diff
@@ -619,8 +754,8 @@ def test_delayed_local_argument():
     trajectory = sampleTrajectory(scenario)
     assert len(trajectory) == 2
     assert len(trajectory[1]) == 2
-    assert tuple(trajectory[1][0]) == (0, 0)
-    assert tuple(trajectory[1][1]) == (-5.5, 12)
+    assert tuple(trajectory[1][0]) == (0, 0, 0)
+    assert tuple(trajectory[1][1]) == pytest.approx((-6, 12, 0), abs=2e-4)
```

Comment:
Why was it necessary to add the absolute tolerance here? There shouldn't be any additional imprecision.